lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb

```diff
@@ -676,6 +676,11 @@ def create_cred(credential_data, domain = '')
   end
 
   def report_creds(type, user, domain, secret)
+    if client.&db_record.id.nil?
```

Comment:
Shouldn't this be `&.` not `.&`? Also should this not also be applied to `id`?

```suggestion
    if client&.db_record&.id.nil?
```

Reply:
Oops. You're 100% right, apologies. 😆